utils_linux.go

```diff
@@ -98,6 +98,13 @@ func getDefaultImagePath(context *cli.Context) string {
 // newProcess returns a new libcontainer Process with the arguments from the
 // spec and stdio from the current process.
 func newProcess(p specs.Process) (*libcontainer.Process, error) {
+	consoleWidth := uint(0)
+	consoleHeight := uint(0)
+	if p.ConsoleSize != nil {
+		consoleWidth = p.ConsoleSize.Width
+		consoleHeight = p.ConsoleSize.Height
+	}
```

Comment:
The defaults of `0` don't make sense given that you unconditionally set the console size when creating it. I don't know what the kernel will do when you tell it to resize a console to 0x0, but from [my quick reading](http://elixir.free-electrons.com/linux/v4.13.4/source/drivers/tty/tty_io.c#L2092) I expect that it isn't a no-op.